### ISSUE TITLE
Use of special characters on website

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-design-system#4024]
This issue is from the accessibility audit of the Design System website by DAC in July 2024.

## DAC's description

* Page: [Accessibility strategy – GOV.UK Design System](https://design-system.service.gov.uk/accessibility/accessibility-strategy/)
* WCAG reference: none (Usability)
* Issue ID: DAC_Special_Characters_Usability_01

There was link in relation to issues found by automated testing tools. Within the link text a tilde (‘~’) was included to represent an approximation of 30% issues found. The inclusion of the tilde could be problematic for some users.

Screen reader users may not interpret the tilde as an approximation symbol. They might read it out as ‘tilde’ or ignore it altogether, leading to misinterpretation of the content to what it was intended.

Also, using the tilde alone does not provide clear context or meaning without additional explanation. Users who are not familiar with its use as an approximation symbol may not understand the intended meaning.

<img width="745" alt="Screenshot of text using the tilde symbol" src="https://github.com/user-attachments/assets/d4d8e851-2fbd-45d2-9aca-cfca552b866f">

## DAC's proposed solution

To ensure clarity and consistency across all users and devices, use the word ‘approximately’
or ‘about’ instead of the tilde to convey the meaning.

## Additional instances

As this can potentially happen on any page, this work should include checking all the other pages other links missing context.
It might be worth making two different cards for this: One for checking and recording all instances, the other for fixing them.

To save time, it makes sense to audit all the body copy issues together:
* #4010
* #4013
* #4023
* #4024
* #4025

## Needed roles

Content designer

```[tasklist]
### Tasks
- [ ] List characters we want to scan for
- [ ] Build process to scan for special characters
- [ ] Run scan
- [ ] Capture report into single document
- [ ] Review document to find instances and decide on actions for each instance
- [ ] Implement actions in PR
- [ ] Review with team
- [ ] Publish
- [ ] Consider adding process to prevent in future
```

**Comment by calvin-lau-sig7** [comment]
For a long while now, I've used this [Deque blog from 2014](https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/) as my reference.

Might be a good starting point for us – or otherwise I'd be keen to update my understanding on this area!

**Comment by calvin-lau-sig7** [comment]
To detect, we can:
- scan the website for the specific special characters

To assess, we can:
- review results and ensure the special characters are appropriate to use, and used in correct way

To resolve, we can:
- correct misused characters
- remove special character and replace with text instead
